WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -468,7 +468,7 @@ import WordPressComAnalytics
         tableView.setContentOffset(CGPoint.zero, animated: true)
     }
 
-    private func propertyForStats() -> [NSObject: AnyObject] {
+    private func topicPropertyForStats() -> [NSObject: AnyObject] {
         assert(readerTopic != nil, "A reader topic is required")
```

Comment:
I know this isn't new code but should we use NSAssert so an exception is throw rather than the app just crashing? nitpick I know :stuck_out_tongue: